apps/emqx_bridge_iotdb/src/emqx_bridge_iotdb_connector.erl

```diff
@@ -610,6 +593,15 @@ on_batch_query(
 on_format_query_result(Result) ->
     emqx_bridge_http_connector:on_format_query_result(Result).
 
+on_add_channel(
+    _InstanceId,
+    _State0,
+    _ChannelId,
+    #{
+        parameters := #{data := []} = _Parameter
+    }
+) ->
+    {error, <<"The data template cannot be empty">>};
```

Comment:
This could be a schema validation as well.

Reply:
I want to allow that old configurations with empty templates do not stop EMQX from starting.